src/gluonts/model/forecast_generator.py

```diff
@@ -118,8 +118,8 @@ def __call__(
                 )
 
             distributions = [
-                self.distr_output.distribution(*u)
-                for u in _extract_instances(outputs)
+                self.distr_output.distribution(distr_args, scale=scale)
+                for distr_args, scale in _extract_instances(outputs)
```

Comment:
I initially had `loc` as well (as you notice it in the comment below) but removed while fixing a test and forgot to add it back. I will add that.

`DistributionForecastGenerator` is not used currently anywhere. This change was necessary because if any model wants to directly output distributions, then their prediction network should return all `dist_args`, `scale`, `loc` so that distribution object can be constructed here by passing them to `distr_output.distribution` function.